src/backend/TrafficCourtsApi/ticket-worker/Features/Emails/EmailSender.cs

```diff
@@ -84,17 +92,20 @@ public async Task SendUsingTemplate(string to, string subject, TicketDisputeCont
 
                 emailModel.LogoImage = dataURIScheme("png", "ticket-worker.Features.Emails.Resources.bc-gov-logo.png");
 
-                var result = await _email
+                var email = _email
                     .To(to)
                     .Subject(subject)
-                    .UsingTemplateFromEmbedded("ticket-worker.Features.Emails.Resources.submissiontemplate.liquid", emailModel, this.GetType().GetTypeInfo().Assembly, true)
-                    .SendAsync();
+                    .UsingTemplateFromEmbedded("ticket-worker.Features.Emails.Resources.submissiontemplate.liquid", emailModel, this.GetType().GetTypeInfo().Assembly, true);
 
-                
-                if (!result.Successful)
+                if ( _emailFilter == null || _emailFilter.IsAllowed(to))
```

Comment:
_emailFilter == null  cannot be null as in contractor, it is already protected it cannot be null.

Reply:
Can we move _emailFilter.IsAllowed to line 91?

Reply:
> Can we move _emailFilter.IsAllowed to line 91?

My intention is to still format the email, even though sending is not allowed.